docs-v2/integrations/all/amazoncognito.mdx

```diff
@@ -0,0 +1,36 @@
+---
+title: Amazoncognito
```

Comment:
@hassan254-prog if this is used for all of AWS APIs, should we maybe call the provider `AWS` instead of `Amazoncognito`?

Reply:
Yes we can rename it to `AWS`. Since `Amazoncognito` serves as a central authentication and authorization service for various `AWS` APIs, using `AWS` as the provider name could indeed provide a clearer representation of its broader functionality.